worker/large_stripe_charge.js

```diff
@@ -0,0 +1,78 @@
+const LUMBERYARD_EVENT = 'mailer';
+const FROM_EMAIL = 'The Mozilla Team <donate@mozilla.org>';
+const TO_EMAIL = process.env.LARGE_DONATION_EMAIL;
+const LARGE_DONATION_AMOUNT = 115;
+// https://support.stripe.com/questions/which-zero-decimal-currencies-does-stripe-support
+const ZERO_DECIMAL_CURRENCIES = [
+  'BIF',
+  'CLP',
+  'DJF',
+  'GNF',
+  'JPY',
+  'KMF',
+  'KRW',
+  'MGA',
+  'PYG',
+  'RWF',
+  'VND',
+  'VUV',
+  'XAF',
+  'XOF',
+  'XPF'
+];
+
+var oxr = require('open-exchange-rates');
+var money = require('money');
+
+oxr.set({ app_id: process.env.OXR_APP_ID });
+
+function update_exchange_rates() {
+  oxr.latest(function() {
+    money.rates = oxr.rates;
+    money.base = oxr.base;
+  });
+}
+
+update_exchange_rates();
+
+// update exchange rates every hour
+setInterval(update_exchange_rates, 1000 * 60 * 60);
+
+module.exports = function(notifier_messager, mailroom) {
+
+  return function(event, cb) {
+
+    if (event.event_type !== 'stripe_charge_succeeded') {
```

Comment:
I'd also add a check to see if `TO_EMAIL` is defined; I imagine we don't want this running on staging